_test/test_tobyfit/test_tobyfit_resfun_2.m

```diff
@@ -0,0 +1,198 @@
+classdef test_tobyfit_resfun_2 < TestCaseWithSave
+    %TEST_TOBYFIT_RESFUN_2 Tests of resolution function plotting
+    
+    properties
+        inst
+        samp
+        det_W
+        det_E
+        det_N
+        det_S
+        ebin
+        ei
+        efix
+        alatt
+        angdeg
+        u
+        v
+        ulen
+        vlen
+    end
+    
+    methods
+        %--------------------------------------------------------------------------
+        function self = test_tobyfit_resfun_2 (name)
+            self@TestCaseWithSave(name);
+            
+            % Make an instrument and sample
+            self.inst = maps_instrument_obj_for_tests(90,250,'s');
+            self.samp = IX_sample(true,[1,0,0],[0,1,0],'cuboid',[0.02,0.02,0.02]);
+            
+            % Make some detectors
+            det.x2=6;
+            det.phi=30;
+            det.azim=0;
+            det.width=0.0254;
+            det.height=0.0367;
+            
+            self.det_W = det;
+            self.det_E = det; self.det_E.azim = 180;
+            self.det_N = det; self.det_N.azim = 90;
+            self.det_S = det; self.det_S.azim = 270;
+            
+            % Some useful parameters
+            self.ebin = [12.5,13.5];
+            self.ei = 100;
+            self.efix = 1;
+            self.alatt = [3,4,5];
+            self.angdeg = [90,90,90];
+            self.u = [1,1,0];
+            self.v = [0,0,1];
+            
+            self.ulen = 2*pi*sqrt(1/self.alatt(1)^2 + 1/self.alatt(2)^2);
+            self.vlen = 2*pi/self.alatt(3);
+            
+            self.save()
+        end
+        
+        
+        %--------------------------------------------------------------------------
+        function test_indepOfSampleOrientation(S)
```

Comment:
Use consistent naming, you used `self` earlier (though I believe `obj` is preferred in MATLAB) and `S` here.

Reply:
I found self.this self.that confusing when I was passing lots of arguments to calls in the tests; I inherited the self. from the template I used to make the test. Happy to change elsewhere to S.

Reply:
Sometimes I just use an unused character like `ŋ` (alt gr+g on my keyboard) and then find and replace with what it's meant to be at the end.